### ISSUE TITLE
Need to connect to Bluehost MySQL - how is it established?

### ISSUE THREAD

**Issue by mshrage-snt-net** [HeidiSQL/HeidiSQL#46]
## Expected behavior

Be able to connect to Bluehost MySQL database for maintenance similar to what phpMyAdmin now permits.

What is the current behavior?

Returns (10060) error

Please help provide information about the failure if this is a bug. If it is not a bug, please remove the rest of this template.

Just need the procedure to access databases running on MySQL on Bluehost shared servers.

Please provide detailed steps for reproducing the issue.

1. step 1
2. step 2
3. you get it...

### Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* HeidiSQL version: latest
* Database system + version:  bluehost latest
* Operating system:  linux (I think)

### Failure Logs

Please include any relevant log snippets or files here.

**Comment by ansgarbecker** [comment]
Please provide more information about the failure.
And please don't ignore the questions. Fill out all items of the issue template.

**Comment by mshrage-snt-net** [comment]
Somehow it doesn't appear to me that you are reading the answers I did provide.  Bluehost seems to have a somewhat non-standard install of MySQL and you might need to establish a trial with them to research this question.  I cannot give all the exact steps without sharing login Id's and passwords that are not appropriate to share.  
HeidiSQL is not connecting to bluehost so I am missing something in the method to use.  I have tried a number of variations of the login and password without success.  Please tell me what you need (excepting the login and password information).  Thank you! Jim

**Comment by alesvaupotic** [comment]
Have you whitelisted your IP with Bluehost?

**Comment by ansgarbecker** [comment]
... and what message comes along with that 10060 error?
Is there some webpage with technical details on Bluehost?

**Comment by mshrage-snt-net** [comment]
OK - thank you for your replies.  I have found the problem - it seems that bluehost requires the connection to be through an SSH Tunnel.  I found that by installing a trial of a different program where the need for tunneling was more apparent in the user interface.  By setting the connection to an SSH tunnel, HeidiSQL is now getting to the database, although it showed an initial error that I clicked through.  Thanks again. Jim
